modules/ROOT/pages/global-settings-configuration.adoc

```diff
@@ -7,24 +7,38 @@ endif::[]
 Global settings for a mule application such as the default transaction timeout or
 default error handler can be configured in the `<configuration>` element of the
 root level of the xref:about-mule-configuration.adoc[Mule Configuration File]. +
-For example:
+
+== Create a Global Configuration from Studio
+
+To create a global settings configuration:
+
+. In Studio, select the *Global Elements* tab and click *Create*:
++
+image::mruntime-global-settings-create.png[create-global-settings]
+. Select *Global Configurations* > *Configuration*:
++
+image:mruntime-global-configurations.png[configure-global-settings]
+. Configure the *Default Error Handler* and the *HA Profile* values.
+
+== Create a Global Configuration by Editing the XML
+
+The following example defines a global error handler, and then a global configuration which specifies a default transaction timeout and references the global error handler as the default one:
 
 [source,xml,linenums]
 ----
 <mule>
   ...
-  <configuration defaultTransactionTimeout="31337" defaultErrorHandler="genericErrorHandler">
-    ...
+  <!-- Error handler -->
+  <error-handler name="GlobalErrorHandler"/>
+  <!-- Global Settings -->
+  <configuration doc:name="GlobalSettings" defaultTransactionTimeout="31337" defaultErrorHandler-ref="GlobalErrorHandler" />
+  ...
   </configuration>
   ...
 </mule>
 ----
 
-== Configuration
-
-Specifies defaults and general settings for the Mule application instance.
-
-=== Attributes of configuration
+== Attributes of <configuration>
```

Comment:
Is this topic and the one following supposed to be a child topic of the task above?